assets/sass/components/_home-content.scss

```diff
@@ -261,34 +246,3 @@
   content: url('/public/images/icons/audio.svg');
   background-color: govuk-colour('green');
 }
-
-.older-content {
-  margin-top: govuk-spacing(5);
-  color: govuk-colour('white');
-  float: right;
-  @include govuk-font(24);
-  text-decoration: none;
-  display: flex;
-  align-items: center;
-  font-weight: 700;
-
-  &:link,
-  &:hover,
-  &:visited {
-    text-decoration: none;
-    color: govuk-colour('white');
-  }
-  &:active,
-  &:focus {
-    color: govuk-colour('black');
-  }
-
-  &::after {
-    background-repeat: no-repeat;
-    content: '';
-    width: govuk-spacing(4);
-    height: govuk-spacing(4);
-    background-image: url('/public/images/icons/chevron-right-white.svg');
-    background-position: 8px 50%;
-  }
-}
```

Comment:
Removed older content CSS and white chevron icon (/public/images/icons/chevron-right-white.svg) that existed in previous design of the Hub.